Source/SwiftLintBuiltInRules/Rules/Idiomatic/MultipleStatementsDeclarationRule.swift

```diff
@@ -0,0 +1,92 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule(explicitRewriter: true, optIn: true)
+struct MultipleStatementsDeclarationRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "multiple_statements_declaration",
+        name: "Multiple Statements Declaration",
+        description: "Statements should not be on the same line",
```

Comment:
We can keep the name a bit shorter:
```suggestion
struct MultipleStatementsRule: Rule {
    var configuration = SeverityConfiguration<Self>(.warning)

    static let description = RuleDescription(
        identifier: "multiple_statements",
        name: "Multiple Statements",
        description: "Every statement should be on its own line",
```
Please also update all other places mentioning the name.